CorsixTH/Lua/cheats.lua

```diff
@@ -73,24 +83,24 @@ function Cheats:cheatResearch()
 end
 
 function Cheats:cheatEmergency()
-  if not self.ui.hospital:createEmergency() then
-    self.ui:addWindow(UIInformation(self.ui, {_S.misc.no_heliport}))
+  if not self.hospital:createEmergency() then
```

Comment:
This isn't something you've introduced, but you can effectively neutralise active epidemics, by starting an emergency here, as the UIWatch check is bypassed at this point. However patients will still be infected/contagious (can't vaccinate them as you fixed this previosuly) well after the natural end, as the 'interrupted' epidemic never finishes.

Reply:
Thanks @mugmuggy , split that out as a separate issue to address later.